### ISSUE TITLE
Intersect between a Point and a GeoJSON file

### ISSUE THREAD

**Issue by carlos-mg89** [tidwall/tile38#341]
Hello,

First of all, I deeply appreciate your work. It's hard to find very well documented information about this topic, that doesn't involve working with a GIS database directly. Or it's for me at least.

I'm trying to perform a reverse Geocoding task where I would like to be certain that a Point is inside a Polygon (based on a multipolygon GeoJSON file, like the ones you get from an Administrative Division from OpenStreetMap in http://polygons.openstreetmap.fr/?id=349053 ).

So let's assume, I have the set of GeoJSON files for the whole world (which is another task and has nothing to do with this). And now I would like to use your project to do so.

I have found this example:

`set city tempe object {"type":"Polygon","coordinates":[[[0,0],[10,10],[10,0],[0,0]]]}`

I guess I could replace what's inside the brackets, with the content of my GeoJSON, but it's too big (several MBs). Would there be a way of setting an object based on a file?

Thanks in advance

**Comment by tidwall** [comment]
Hi, 

Unfortunately there isn't a way to directly SET or GET an object from a file using a Tile38 command. You will need to directly load the content of the file into a variable and then send Tile38 a SET command with the variable.

For example in Go using the Redigo client library:

```go
data, _ := ioutil.ReadFile("tempe.geojson")
conn.Do("SET", "city", "tempe", "OBJECT", string(data))
```

Now if you have a bunch of points in a collection, let's call it "people", and you want to see if they are in Tempe:

```
INTERSECTS people GET city tempe
```

Let me know if I'm off track here.

**Comment by carlos-mg89** [comment]
At the end I managed to accomplish this task by using a Point in polygon intersection algorithm implemented in this Git repository https://github.com/pelias/polygon-lookup

However, thank you very much for answering as fast as you did. I totally forgot to reply back.

Your proposal seemed reasonable to start with, although this other project I found was a bit more straight forward, and I needed to save some development time.
